### DISCUSSION TITLE
关于网站的需求和选型

### DISCUSSION THREAD

**Discussion by hexWars** [fp-space/MiniNote#1] [Ideas]
目前是说要用Actix是吧，但是据我所知Actix是后端框架，但是之前说的是静态网站，所以怎么说

界面部分，用tailwindcss？然后vue？组件框架呢

> 我个人推荐是 tailwindcss + shadcn  一套梭哈，简单方便，用vue还是react都行

@LordFoxFairy

**Comment by LordFoxFairy** [comment]
1. 语言选定rust
2. 采用成熟的框架 Yew
是否赞同 @hexWars

**Reply by hexWars** [reply]
> 1. 语言选定rust
> 2. 采用成熟的框架 Yew
>    是否赞同 @hexWars

赞同，可以使用yew，下面是一些可能有用的链接：

- https://github.com/jetli/awesome-yew
- https://github.com/sheshbabu/rustmart-yew-example
- https://github.com/stkevintan/Kirk
- https://github.com/PepcyCh/pepcy-site
- https://github.com/zzhack-stack/zzhack

css框架就直接用tailwindcss吧，这玩意很简单可以看文档直接上手，先按阶段实现吧

第一阶段：有个顶部的导航栏，页面中间可以显示文字和图像

第二阶段再解析markdown

@LordFoxFairy

**Reply by LordFoxFairy** [reply]
@hexWars css 使用 yew-style，对应的前端框架都有yew替代的框架可以引入

**Reply by LordFoxFairy** [reply]
1. 划分内容板块
2. 规划设计布局，方便以后的样式快速替换

**Reply by hexWars** [reply]
`yew-style` 我不是很清楚，可以试试

版块具体如何设计，左边导航栏还是顶部导航栏？

**Reply by LordFoxFairy** [reply]
布局先按照左边目录树，顶部导航栏形式吧，右下内容区域这种形式吧

**Reply by hexWars** [reply]
可

**Comment by hexWars** [comment]
关于分工问题

你/我把要做的可以做的东西，放到issue里，分别认领一部分来写，你觉得这种协调方式怎么样

@LordFoxFairy

**Reply by LordFoxFairy** [reply]
我觉得可以，目前是先设计下需求吧，1. 要做什么， 2. 尽量简洁， 3. 针对样式，一定可以方便修改，所以样式也放在固定文件下
